jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -12,87 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Furnishings job processing rules after stage runs of involuntary dissolution."""
+import os
 from datetime import datetime
+from io import StringIO
 from pathlib import Path
 from typing import Final
 
 from flask import Flask, current_app
 from jinja2 import Template
 from legal_api.models import Furnishing, FurnishingGroup, XmlPayload
 from legal_api.utils.legislation_datetime import LegislationDatetime
+from paramiko.ssh_exception import AuthenticationException, NoValidConnectionsError
 
+from furnishings.sftp import SftpConnection
 
-XML_DATE_FORMAT: Final = '%B %-d, %Y'
-
-
-class PostProcessor:
-    """Processor after stage run of furnishings job."""
```

Comment:
Move the `PostProcessor` class below the `XMLMeta` class so it can be closer to the `process` function.